WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -15,8 +15,13 @@
 import org.wordpress.android.ui.reader.utils.ReaderUtils;
 import org.wordpress.android.ui.stats.StatsTimeframe;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class AppPrefs {
     private static final int THEME_IMAGE_SIZE_WIDTH_DEFAULT = 400;
+    private static final int MAX_PENDING_DRAFTS_AMOUNT = 100;
```

Comment:
Well the only place where we are deleting ids from the ignore list pref is when the draft is trashed/deleted. We should also delete ids (if found) when a draft is published (we aren't atm, will make the call in a next commit). But there's also the possibility case where for some reason ids can't be deleted one way or the other (let's call it random failure) and ids can be kept in that list forever (well, at least as long as you don't switch sites, or when you are an admin for one site only). I wouldn't want the ignore id list to grow indefinitely, so that's why this precautious method is there. A comment on that, 100 seemed a large enough number for this functionality, we could lower it to 15 or 20 and it'll pretty much be the same I think.